src/components/date/date.test.jsx

```diff
@@ -89,7 +89,7 @@ describe('src/components/date/date.test.jsx', () => {
       render(<Date {...requiredProps} />);
       userEvent.type(screen.getByLabelText('Test label'), '');
       expect(screen.getByText(today.calendarHeading)).toBeInTheDocument();
-      userEvent.type(screen.getByLabelText('Test label'), '{esc}', { skipClick: true });
```

Comment:
Ah, yeah, I think this is what I was talking about in slack - I knew there was a _way_ to do keypresses, but `keyboard` is definitely more intuitive